src/test/java/com/google/javascript/gents/singleTests/access_modifiers.ts

```diff
@@ -10,10 +10,15 @@ class A {
   private e: number;
   // Method access
```

Comment:
I'm curious if there is an easy way to add a space between fields and the constructor as well - what if you added before rather than after the method body? No need to hold up this PR if it isn't straightforward.